src/main/java/org/prnhs/javaee/swim/web/ProgramController.java

```diff
@@ -53,7 +64,13 @@ public ProgramDto getById(@PathVariable Integer id){
     @ApiResponse(code = 200, message ="OK", response = ProgramDto.class)})
     public List<ProgramDto> getAll(){
         LOGGER.debug("getAll method called in ProgramController class");
-        return programService.getAll();
+        
+        List<ProgramDto> programs = programService.getAll();
+        programs.stream().forEach((program) -> {

```

Comment:
i wrote a for each loop to iterate, and when i switched to stream with netbeans, it added those brackets by itself